README.md

```diff
@@ -1,77 +1,119 @@
 # Skype Sync SDK
 
-Skype Sync allows you create shared experiences across different Skype Interviews instances.
+Skype Sync allows you create shared experiences across different Skype Interviews instances. You can use Skype Sync to synchronize experience between sessions.
 
 ## Getting started
 
 Let's assume we are building a simplified synchronized note pad app.
 
-1. Import the Sync SDK
-```js
-import Sync from 'skype-sync'
+1. Install skype-sync npm package.
+```
+npm install skype-sync
 ```
 
-2. Make sure to setup initialization handlers to accomondate for your UI.
-```js
-Sync.init((configuration) => {
-  // initialize your application with given configuration
-})
+2. Import the Sync SDK
+```ts
+import Sync, { InitContext } from 'skype-sync';
+```
+
+3. Make sure to setup initialization handlers to accomondate for your UI.
+```ts
+Sync.initHandler = this.syncSdkReady.bind(this);
+```
+
+4. Implement the initialization handler. It will be called when Skype Sync is ready to be used, it will also contain Initialization Context.
```

Comment:
Simplification: It will be called when Skype Sync is ready to be used and will contain InitializationContext.